### ISSUE TITLE
Implement OpenRGB support

### ISSUE THREAD

**Issue by JohnAZoidberg** [FrameworkComputer/inputmodule-rs#18]
WIP at https://github.com/FrameworkComputer/OpenRGB

**Comment by khuongduybui** [comment]
404'ed. Is that a private repo for now?

**Comment by CalcProgrammer1** [comment]
Is this for the upcoming Framework 16?  I want to get my hands one one of these for OpenRGB keyboard (and hopefully LED matrix) support.

**Comment by JohnAZoidberg** [comment]
> 404'ed. Is that a private repo for now?

Yes. Actually I should delete that, move it over to a fork on GitLab and eventually open a merge request there.

> Is this for the upcoming Framework 16? I want to get my hands one one of these for OpenRGB keyboard (and hopefully LED matrix) support.

Yes it is! This repository here is for the white LED Matrix module (and other non-keyboard input modules).
Couldn't take advantage of all of OpenRGB but I think it wouldn't be hard to add.

> If the keyboard is using QMK, then I would wait for the QMK people to finalize the XAP protocol including the direct mode.

It is. Check out the current work-in-progress fork at https://github.com/FrameworkComputer/qmk_firmware/pulls
Okay good, I thought so too. Let's see if it can be ready before our launch.
But even if not, the firmware is easily updateable by users.

**Comment by CalcProgrammer1** [comment]
Is Framework considering RGB matrix panels?  I'd really like to see that
option!
